### ISSUE TITLE
Can't set external storage

### ISSUE THREAD

**Issue by Bables55** [nextcloud/server#9699]
### Steps to reproduce
1. Go Accout > Settings
2. Select External Storage from left pane.
3.

### Expected behaviour
The already set list is displayed.

### Actual behaviour
Displayed "No external storage configured or you don't have the permission to configure them"

Next, when I disable the External Storage application, Nextcloud stops. And I can change the value of files_external from No to Yes in phpMyAdmin and restart Nextcloud. After that I have lost the item of external storage in the left pane when I look at the settings. It seems that actual external storage handling is done in this state. In other words, storage can not be set up.

### Server configuration

**Operating system**:
CentOS 7.5
**Web server:**
nginx 1.13.8
**Database:**
MariaDB 5.5.56
**PHP version:**
PHP 7.1.18
**Nextcloud version:** (see Nextcloud admin page)
13.0.2
**Updated from an older Nextcloud/ownCloud or fresh install:**
Nextcloud 13.0.1
**Where did you install Nextcloud from:**
nextcloud.com
**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
  - activity: 2.6.1
  - admin_audit: 1.3.0
  - announcementcenter: 3.2.1
  - apporder: 0.4.1
  - bruteforcesettings: 1.0.3
  - calendar: 1.6.1
  - comments: 1.3.0
  - contacts: 2.1.3
  - dav: 1.4.6
  - deck: 0.3.1
  - drawio: 0.8.9
  - encryption: 2.0.0
  - end_to_end_encryption: 1.0.5
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_accesscontrol: 1.3.0
  - files_antivirus: 1.2.0
  - files_automatedtagging: 1.3.0
  - files_external: 1.4.1
  - files_frommail: 0.1.0
  - files_markdown: 2.0.4
  - files_pdfviewer: 1.2.1
  - files_rightclick: 0.8.4
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - gallery: 18.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - mail: 0.8.1
  - mindmaps: 0.1.0
  - nextcloud_announcements: 1.2.0
  - notes: 2.3.2
  - notifications: 2.1.2
  - oauth2: 1.1.0
  - password_policy: 1.3.0
  - provisioning_api: 1.3.0
  - quicknotes: 0.1.3
  - ransomware_protection: 1.1.0
  - richdocuments: 2.0.8
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - sharepoint: 1.1.0
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - tasks: 0.9.6
  - theming: 1.4.5
  - twofactor_backupcodes: 1.2.3
  - updatenotification: 1.3.0
  - user_external: 0.4
  - workflowengine: 1.3.0
Disabled:
  - firstrunwizard
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
    "system": {
        "debug": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "MY-DOMAIN",
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/MY-DOMAIN",
        "dbtype": "mysql",
        "version": "13.0.2.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "log_type": "file",
        "logtimezone": "Asia\/Tokyo",
        "loglevel": 2,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "filelocking.enabled": true,
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpsecure": "ssl",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "auth.bruteforce.protection.enabled": false,
        "versions_retention_obligation": "7,auto",
        "trashbin_retention_obligation": "auto",
        "defaultapp": "calendar",
        "theme": ""
    }
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
SFTP
**Are you using encryption:** yes/no
Yes
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
WebDAV

### Client configuration
**Browser:**
Firefox, Vivaldi
**Operating system:**
Windows 10
#### Web server error log
<details>
<summary>Web server error log</summary>

```
2018/05/31 11:33:29 [error] 2699#0: *6 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527734009920 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
2018/05/31 11:33:34 [error] 2699#0: *6 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527734014087 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
2018/05/31 11:34:13 [error] 2699#0: *6 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527734053682 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
2018/05/31 11:49:24 [error] 12734#0: *419 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527734964588 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
2018/05/31 11:49:42 [error] 12734#0: *419 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527734982637 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
2018/05/31 11:49:51 [error] 12734#0: *419 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527734991007 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
2018/05/31 11:59:50 [error] 12734#0: *419 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527735590720 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
2018/05/31 12:17:59 [error] 12734#0: *419 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527736679023 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
2018/05/31 12:18:04 [error] 12734#0: *419 access forbidden by rule, client: MY-GLOBAL-IP, server: MY-NEXTCLOUD-DOMAIN, request: "GET /data/.ocdata?t=1527736684688 HTTP/2.0", host: "MY-NEXTCLOUD-DOMAIN"
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Error	core	Can't get app storage, app files_external, user not logged in	
```
</details>

**Comment by Bables55** [comment]
I put files_external in the Nextcloud 13.0.1 package into the current Nextcloud. And I accessed Nextcloud with Web browser and processed the update. Thereafter, "External Storage" was displayed above and below the left pane of the setting.
 I came to be able to use it. I noticed that this time I tried to add external storage for the first time in a while. I don't know if it was my only problem or coincidence.
